### PR TITLE
Wiremock emb usecases

### PR DESCRIPTION
**Pull request by seran**

### CODE REVIEW

File: docs/loopback.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Multiple loopback interface for tests
+
+WireMock tests require different loopback address other than 127.0.0.1.
+
+By default, in macOS, this address is not available to enable it run 
+the following command.
+
+`sudo ifconfig lo0 alias 127.0.0.2`
```
Comment (arcuri82) [inline docs/loopback.md:8]: does this need to be done for every single 127.x.x.x address?
in that case, might want to create a script to do that (eg a loop over all the 256^3 addresses)

File: client-java/instrumentation/src/main/resources/keepInstrumentationList.txt [hunk 2]
```diff
@@ -1,4 +1,3 @@
 //list of classes/packages to include regardless of skipList.
 //this is useful when you want to ban a domain org.foo but still allow
 //some of its subpackages, like org.foo.bar
-
```
Comment (arcuri82) [inline client-java/instrumentation/src/main/resources/keepInstrumentationList.txt:4]: any reason for modyfing this file?

**Review by arcuri82 — CHANGES_REQUESTED** [review]

File: docs/loopback.md (hunk 1 above)
Comment (arcuri82) [reply docs/loopback.md:8]: note: can just leave a comment about it. no need to write the script for this PR

File: client-java/instrumentation/src/main/resources/keepInstrumentationList.txt (hunk 2 above)
Comment (seran) [reply client-java/instrumentation/src/main/resources/keepInstrumentationList.txt:4]: Did some experiments, previous commit captured the changes. In this reverted it back to the original version.

File: docs/loopback.md (hunk 1 above)
Comment (seran) [reply docs/loopback.md:8]: I'll add the script in the next commit.

**Pull request merged**